src/Razor/test/RazorLanguage.Test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -2905,7 +2905,7 @@ public void EventHandler_PreventDefault_StopPropagation_Minimized()
             // Act
             var generated = CompileToCSharp(@"
 @using Microsoft.AspNetCore.Components.Web
-<input @onclick:preventDefault @onclick:stopPropagation />");
+<button @onclick:preventDefault @onclick:stopPropagation>Click Me</button>");
```

Comment:
Is the feature not supposed to work for `input type=button` ?

Reply:
It works for input. But inputs don't typically have child content ("Click Me") so I changed it to a button